### ISSUE TITLE
Strange slowdown when open to edit a .htaccess file.

### ISSUE THREAD

**Issue by Karklis** [adobe/brackets#11040]
With livereload running, apache running, when open .htaccess to edit in brackets, the IDE became too slow.

.htaccess file downloaded from: https://github.com/h5bp/html5-boilerplate/blob/master/dist/.htaccess

- windows 7 64 ultimate;
- With or without apache 2.4 (XAMPP) running;
- Brackets Release 1.3 build 1.3.0-16022 (release cd0a6aae5) build timestamp: Wed Apr 29 2015 11:07:50 GMT+0100 ;
- with or without livereload
- chrome 42 (64bit)
- With or without "apache syntax" brackets extension

**Comment by abose** [comment]
Does this problem persist after restarting brackets?
Can you try `debug>reload without extensions` and see if the slowdown persists?

**Comment by Karklis** [comment]
yes, slowdown persists, but became a little faster... but I still having lags scrolling... with sublimetext everything is ok, but brackets still slow...

**Comment by abose** [comment]
Did you try Can you try `debug>reload without extensions`

**Comment by Karklis** [comment]
yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes.

**Comment by abose** [comment]
:sweat_smile:  I can't see the slowness in my machine.
What is the project size you opened? If you open the file from the Default 'Getting Started` project, and open the .htaccess file, is it slow?
Could you see any messages in the developer console?
Also What is the ram/processor utilization you are seeing in the task manager.
